### PR TITLE
Adding guidance for function app

### PR DESCRIPTION
**Pull request by xiaxyi**

### CODE REVIEW

**Comment by xiaxyi** [comment]
updated config:

result:
```
No changes. Your infrastructure matches the configuration.
```

File: quickstart/201-function-app/main.tf [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+resource "azurerm_linux_function_app" "test" {
+  name                       = "${var.name_prefix}-lfa"
+  location                   = azurerm_resource_group.default.location
+  resource_group_name        = azurerm_resource_group.default.name
+  service_plan_id            = azurerm_service_plan.default.id
+  storage_account_name       = azurerm_storage_account.default.name
+  storage_account_access_key = azurerm_storage_account.default.primary_access_key
+  https_only                 = true
+  builtin_logging_enabled    = false
+  functions_extension_version = "~4"
```
Comment (lonegunmanb) [inline quickstart/201-function-app/main.tf:39]: Could we run `terraform fmt` for this folder to fix this format issue?

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Thanks @xiaxyi for opening this pr, almost LGTM except one tiny format issue here.

File: quickstart/201-function-app/main.tf (hunk 1 above)
Comment (xiaxyi) [reply quickstart/201-function-app/main.tf:39]: Thansk @lonegunmanb , updated~

**Review by lonegunmanb — COMMENTED** [review]
Thanks @xiaxyi, Terraform's part LGTM, but still needs @TomArcherMsft for further review.

**Pull request merged**